### DISCUSSION TITLE
如果src 和tgt 都是本地文件系统，同步的速度会超过rsync吗？

### DISCUSSION THREAD

**Discussion by qiuwei** [juicedata/juicesync#113] [General]
具体使用场景是将大量小文件从一个磁盘 sync到另外一个磁盘。
不知道使用juicesync 相对于rsync 会有提速吗？ 
使用rsync时明显没有将磁盘的读取、写入带宽用满。

另外可以解释下在sync时，terminal上显示的信息具体代表什么含义吗？
![image](https://user-images.githubusercontent.com/1304505/120074455-95dd3e80-c0cf-11eb-9dba-2ec5dfeab3d7.png)

**Comment by davies** [comment]
juicesync 默认使用 20 个线程来拷贝数据，估计相比 rsync 的单线程会有一定的性能提升。

左边部分是进度条，= 表示没有变化的比例，+ 表示新拷贝的比例， 空格表示发现了但是尚未拷贝的文件数。右边的数字依次是 发现的总文件数、当前已经完成的比例（包括不需要拷贝的）、每秒钟拷贝的文件数、每秒钟拷贝的数据量。
